### DISCUSSION TITLE
Getting the Return or Frame Address of a Function

### DISCUSSION THREAD

**Discussion by ghost** [LADSoft/OrangeC#1015] [General]
GCC supports this feature: https://gcc.gnu.org/onlinedocs/gcc/Return-Address.html

I only want to ask if Orange C already supports this feature and whether you want to support it or not.

**Comment by LADSoft** [comment]
it is sometimes useful to do so, I think the runtime library sometimes codes assembly code to get such addresses but that is kind of a kluge.    I will think about it....
